src/components/SideMenu/styled.elements.ts

```diff
@@ -112,7 +123,10 @@ export const MenuItemLabel = styled.label.attrs((props: MenuItemLabelType) => ({
   ${props => (props.deprecated && deprecatedCss) || ''};
 
   &:hover {
```

Comment:
Just noting here that the "active" styling of the link that you've made is slightly different from the current stylings. HOWEVER, I actually prefer yours. I'll attach the two screenshots, for posterity. I think yours is more pleasing and simple.
![Screen Shot 2022-12-07 at 5 01 32 PM](https://user-images.githubusercontent.com/22421112/206306749-177aedaf-e295-4a46-b65f-fbeb863619e7.png)
![Screen Shot 2022-12-07 at 5 01 51 PM](https://user-images.githubusercontent.com/22421112/206306750-4a0302de-a9da-4fa6-9f7a-77777b07000f.png)

Reply:
Good catch! I think that extra second line in the first (snooty) screenshot is due to inherited styling from LeafyGreen's `Link` component. The extra height of the back button in the second screenshot is due to reusing Redoc's existing menu components. I can reach out to Allison to see what the ideal style here is

Reply:
Spoke with Allison about changing only the padding and margins of things. Latest changes should be okay!